### ISSUE TITLE
Add TabControl

### ISSUE THREAD

**Issue by skendrot** [Microsoft/xaml-standard#30]
All platforms have a way of displaying tabs. XAML Standard needs a TabControl as well

**Comment by bschoepke** [comment]
UWP calls this "[Pivot](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Pivot)"

Shorter and sweeter.

**Comment by skendrot** [comment]
Yes, but Pivot really doesn't make sense on other platforms. I believe TabControl is a proper name and is the name used in WPF as well

**Comment by Odonno** [comment]
Indeed, it is a common control shared between all 3 major mobile platforms. In Windows 10, I think the TabControl of Microsoft Edge can be reused.

**Comment by birbilis** [comment]
Pivot can be confused with PivotViewer btw

**Comment by mdtauk** [comment]
TabControl is a standard across platforms, so should be included.  But UWP implementations of XAML Standard could map it to the Pivot, or build a TabControl built on top of the Pivot.  Like how NavigationView is built on SplitView

**Comment by weitzhandler** [comment]
`TabControl` should be there. Dislike `Pivot`, it means nothing outside UWP.
